invenio_stats/aggregations.py

```diff
@@ -255,18 +265,24 @@ def _get_oldest_event_timestamp(self):
         """Search for the oldest event timestamp."""
         # Retrieve the oldest event in order to start aggregation
         # from there
-        query_events = dsl.Search(using=self.client, index=self.event_index).sort(
-            {"timestamp": {"order": "asc"}}
-        )[0:1]
+        query_events = (
+            dsl.Search(using=self.client, index=self.event_index)
+            .sort({"timestamp": {"order": "asc"}})
+            .extra(size=1)
+        )
         result = query_events.execute()
         # There might not be any events yet if the first event have been
         # indexed but the indices have not been refreshed yet.
         if len(result) == 0:
             return None
+        print(
```

Comment:
I took the liberty of printing some warnings  for the task. This one in particular should be printed only once. They can be removed if you think that it polutes too much